### DISCUSSION TITLE
"A Javascript error occured" after note deletion

### DISCUSSION THREAD

**Discussion by Potjoe-97** [TriliumNext/Trilium#2486] [Q&A]
Hi, 

I was manipulating some notes with image linked into them, and deleted one of the image note from one embedded in another note. Since then, I get popup every 5-10 seconds indicating a Javascript error : 
![Javascript Error](https://user-images.githubusercontent.com/42873357/147581252-dea86f18-b4bb-4a4d-bc8f-720847e29d83.png)
Is there a way to fix the issue without having to restore a backup ?

Thank you for your help!!

**Comment by zadam** [comment]
Hi, I think the error should disappear once you restart the application.
